src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -93,6 +97,7 @@ export class ExamParticipationComponent implements OnInit, OnDestroy {
             this.examId = parseInt(params['examId'], 10);
             this.examParticipationService.loadExam(this.courseId, this.examId).subscribe((exam) => {
                 this.exam = exam;
+                this.endDate = exam.endDate ? exam.endDate : moment();
```

Comment:
Should you maybe use the server time here instead of moment()? Seems to be an edge case anyway as exam.endDate should be set anyway

Reply:
Good catch, I changed it now.